internal/util/debug/debug.go

```diff
@@ -20,38 +20,60 @@ import (
 	"context"
 	_ "expvar" // for metrics
 	"fmt"
+	"log"
+	"math/rand"
+
 	"net"
 	"net/http"
 	_ "net/http/pprof" // for profiling
 	"slices"
 	"text/template"
 	"time"
 
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/arl/statsviz"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	io_prometheus_client "github.com/prometheus/client_model/go"
 	"go.uber.org/zap"
 	"golang.org/x/exp/maps"
-
-	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+type metric struct {
+	metricName       string
+	metricType       int
+	metricLabel      string
+	metricLabelValue string
```

Comment:
I'm not sure how we fit multiple key/value pairs into just two fields. I'm also not sure why we need that structure at all

Reply:
We don't fit Multiple key/value pairs because it's not possible to fit more than one Timeseries into a single Plot programatically, hence each graph plot only represents a unique single key/value pair even if metricFamily.Name is simillar across them. We need that structure because the GetValue attribute of statsviz.Timeseries doesn't support passing arguments (in this use-case passing metricFamily.Name field). therefore creating a structure would allow data passing thru interface.